pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -5,18 +5,22 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/percona/percona-xtradb-cluster-operator/version"
+
 	v "github.com/hashicorp/go-version"
 	"github.com/pkg/errors"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
-	k8sversion "k8s.io/apimachinery/pkg/version"
 )
 
+const WorkloadSA = "percona-xtradb-cluster-operator-workload"
+
 // PerconaXtraDBClusterSpec defines the desired state of PerconaXtraDBCluster
 type PerconaXtraDBClusterSpec struct {
-	Platform              Platform                             `json:"platform,omitempty"`
+	Platform              version.Platform                     `json:"platform,omitempty"`
+	APIVersion            string                               `json:"apiVersion,omitempty"`
```

Comment:
sorry about that, can we rename it a bit
```suggestion
	CRVersion            string                               `json:"CRVersion,omitempty"`
```